governance/polls/Extend Strategic Happiness Core Unit (SH-001) Budget Until May Governance Cycle - April 25, 2022.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Extend Strategic Happiness Core Unit (SH-001) Budget Until May Governance Cycle - April 25, 2022
+summary: Signal your support or opposition to sending a one-time transfer of 35,000 DAI to the Strategic Happiness Core Unit
```

Comment:
Full stop at the end of the sentence please

Reply:
Can you also include SH-001, so "Strategic Happiness Core Unit (SH-001)." please?